importer/importer/builder.py

```diff
@@ -389,6 +391,42 @@ def save_image(image_source_url):
         return 0
 
 
+def get_product_accountability_period(product_id: str) -> int:
+    product_endpoint = "/".join([fhir_base_url, "Group", product_id])
+    response = handle_request("GET", "", product_endpoint)
+    if response[1] == 200:
+        json_product = json.loads(response[0])
+        product_characteristics = json_product["characteristic"]
+        for character in product_characteristics:
+            if (
+                character["code"]["coding"][0]["display"]
+                == "Accountability period (in months)"
+            ):
+                accountability_period = character["valueQuantity"]["value"]
+                return accountability_period
+        logging.error(
+            "Accountability period was not found in the product characteristics : "
+            + product_id
+        )
+        return -1
+    else:
+        logging.error(
+            "Error while attempting to get the accountability period from product : "
+            + product_id
+        )
+        logging.error(response[0])
+        return -1
+
+
+def calculate_date(delivery_date: str, product_accountability_period: int) -> str:
+    delivery_datetime = datetime.strptime(delivery_date, "%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
@peterMuriuki is this date format assumption correct? or should we make this a parameter?

Reply:
yeah that looks correct, `The format is YYYY, YYYY-MM, YYYY-MM-DD or YYYY-MM-DDThh:mm:ss+zz:zz,`

ref: https://www.hl7.org/fhir/R4/datatypes.html#dateTime

Reply:
@peterMuriuki I was thinking more inline of the csv. Since those are not necessarily coming from the server
Unless someone exports the resources. 
I'll create a separate issue to extend this just in case